packages/apps/job-launcher/server/package.json

```diff
@@ -9,7 +9,8 @@
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest start",
-    "start:dev": "nest start --watch",
+    "start:watch": "nest start --watch",
```

Comment:
Remove start:watch. We can just use start:dev, you can remove this NODE_ENV=development and call it as:
NODE_ENV=development yarn start:dev